testdata/configtest.json

```diff
@@ -1,5 +1,6 @@
 {
  "name": "Skynet",
+ "dataDirectory": "",
```

Comment:
this file shouldnt be changed it upgrades internally. can u revert it back to where master is pls.

Reply:
done